index.html

```diff
@@ -531,103 +531,254 @@ <h2>
 
       <section class="normative">
         <h2>
-  Operations
+  Operations and Authorization Structure
         </h2>
 
+        <p class="issue" data-number="134"></p>
+
+        <p class="issue" data-number="138"></p>
+
+        <p class="issue">Does it make sense to differentiate 'create' from 'update',
+        in terms of allowed actions, or is 'write' sufficient?</p>
+
         <p>
 Data vaults allow for the classic set of CRUD (Create, Read, Update and Delete)
 operations on its data models.
+All operations perfomed by an Encrypted Data Vault must carry appropriate
+authorization. This section considers the structure (data model) of
+authorizations in the context of operations they're authorizing.
         </p>
 
         <section class="normative">
           <h3>
     Create Operation
           </h3>
 
-          <ul>
-            <li>
-Create Vault: Creates/provisions an Encrypted Data Vault by specifying a
-<code>DataVaultConfiguration</code> structure. This includes Replication configuration.
-            </li>
-            <li>
-Create Index: Creates an Index for a particular Vault.
-            </li>
-            <li>
-Create Resource (Document or Stream): Creates a Resource in a given Vault.
-            </li>
-          </ul>
+          <h4>Create Vault</h4>
+
+          <p>
+This operation creates/provisions an Encrypted Data Vault instance by specifying
+a <code>DataVaultConfiguration</code> structure (see the Encrypted Data Vault
+Data Model section). This includes Replication configuration.
+          </p>
+
+          <pre class="example highlight"
+            title="Example authorization capability for a Create Vault operation.">
+{
+  "@context": ["https://w3id.org/security/v2"],
+  "allowedAction": "write",
+  // entity on which the operation is performed
+  "invocationTarget": "https://example.com/edvs",
+  // cryptographic key which is allowed to perform the operation
+  "invoker": "did:example:abcd#key1",
+  "proof": {
+    // ...
+  }
+}
+          </pre>
+
+          <h4>Create Resource</h4>
+
+          <p>
+This operation creates an encrypted resource (Document or Stream), in a given
+vault instance.
+          </p>
+
+          <pre class="example highlight"
+            title="Example authorization capability for a Create Resource operation.">
+{
+  "@context": ["https://w3id.org/security/v2"],
+  "allowedAction": "write",
+  // entity on which the operation is performed
+  "invocationTarget": "https://example.com/edvs/z4sRgBJJLnYy/docs",
```

Comment:
Note: The current implementations spell out `documents` in the HTTP API.

```suggestion
  "invocationTarget": "https://example.com/edvs/z4sRgBJJLnYy/documents",
```